Stratis.Bitcoin/Configuration/NodeArgs.cs

```diff
@@ -147,8 +151,9 @@ public static NodeArgs GetArgs(string[] args)
 			}
 			nodeArgs.Testnet = args.Contains("-testnet", StringComparer.CurrentCultureIgnoreCase);
 			nodeArgs.RegTest = args.Contains("-regtest", StringComparer.CurrentCultureIgnoreCase);
+			nodeArgs.Prune = int.Parse(args.Where(a => a.StartsWith("-prune=")).Select(a => a.Substring("-prune=".Length).Replace("\"", "")).FirstOrDefault() ?? "0");
```

Comment:
Use the `config` variable below for getting the value, so prune can be used both with command line or config file.